### ISSUE TITLE
Is there anyway to integrate the plugin with cmp for autocomplete suggestions?

### ISSUE THREAD

**Issue by MarcoCamilo** [nvim-telescope/telescope-bibtex.nvim#61]
At the moment I'm using this plugin, since it's the only one that I've found to flawlessly work with Better BibTeX citation keys. However, I find the need to open Telescope each time I want to add a citation key to add friction to the workflow.

I was wondering if there's anyway currently to integrate the plugin with cmp, so that I can get autocomplete suggestions based on the results from this plugin.

**Comment by noahares** [comment]
I do not think there is a way currently. From what I know, you need to explicitly provide cmp with the items it should display.
Because this is a plugin for telescope, that's beyond the scope of this project.
Maybe use the bibtex LSP? It basically does what I assume you need: Typing "\cite{" opens "Operator" completion items with a small preview of the entries. You can then type pretty much any field of the entry you want and it finds the correct cite key.

**Comment by MarcoCamilo** [comment]
Yeah, the problem is when I'm using another syntax, like Markdown, where I use Pandoc citation keys such as `[@citation]`, then the autocomplete doesn't work. I thought somehow one could export the items provided by the plugin and feed them to cmp (maybe too imaginative on my part xD).

Eitherways, thanks for the feedback :)

**Comment by noahares** [comment]
Yeah, sorry, this would indeed be nice, but probably something for another plugin. This one does too much already.
